mix.lock

```diff
@@ -1,20 +1,20 @@
-%{"bunt": {:hex, :bunt, "0.2.0", "951c6e801e8b1d2cbe58ebbd3e616a869061ddadcc4863d0a2182541acae9a38", [:mix], [], "hexpm"},
```

Comment:
New version of Hex?

Reply:
More like an older version. @theodowling Please exclude the mix.lock change :) You might want to upgrade with `mix local.hex`

Reply:
Thanks for the heads-up. I've reverted change to `mix.lock`